tests/test_version.py

```diff
@@ -1,7 +1,10 @@
 from __future__ import annotations
 
+from tox import __version__
+from tox.pytest import ToxProjectCreator
 
-def test_version() -> None:
-    from tox import __version__
 
-    assert __version__
```

Comment:
You'll need to add a test that validates the output with plugins. Check some of our tests that validate the plugin system to find out how.